### ISSUE TITLE
suggest adding more documentation in Sessions/Session8/DeepLearning_Segmentation.ipynb

### ISSUE THREAD

**Issue by guillepinto** [semilleroCV/Semillero#8]
I noticed that the data loading, visualization, dataset and dataloader sections seem to be based on a Kaggle notebook ([link](https://www.kaggle.com/code/chihjungwang/retina-segmentation-by-pytorch-f1-score-80)). However, some important topics, such as the explanation of the UNET model and the training, evaluation and inference loops, are not detailed.

If it is OK with you, I could propose an improved version that includes additional documentation and references to tutorials that could be useful to develop the notebook more completely.

**Comment by Factral** [comment]
of course, thanks for the initiative! I agree that more documentation is needed.

add the information you see important and add me as a reviewer in the PR.
